ui_components/src/home/Home.js

```diff
@@ -559,6 +511,36 @@ function NewsSection({ newsRef, newsPosts, homeLinks }) {
         return formattedDate;
     }
 
+    function cleanNewsContents(contents) {
+        console.log(contents);
+        const parser = new DOMParser();
+        const decodedString = parser.parseFromString(contents.replace(/[\r\n]+/g, " ").replace(/<br\s*\/?>/g, '\n'), 'text/html').body.textContent;
+        return decodedString;
```

Comment:
Can you add a comment about what the intent of this is?

Reply:
Per discord convo.